### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by colin-higgins**
Changed README to a short blurb with relevant links

cc / @kbaley @mauroservienti @DavidBoike

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
 
 ```csharp
 var documentStore = new DocumentStore 
 { 
     Url = "http://myravendb.mydomain.com/" 
 };
-configure.UsePersistence<RavenDB>(_ => _.SetDefaultDocumentStore(documentStore));
 documentStore.Initialize();
+var asyncSession = documentStore.OpenAsyncSession();
+configure.UsePersistence<RavenDB>().UseSharedAsyncSession(() => asyncSession);
```
Comment (colin-higgins) [inline README.md]: Forcing them to pass an IAsyncDocumentSession kind of sucks

File: README.md [hunk 2]
```diff
@@ -32,9 +35,7 @@ Both approaches will use the same DocumentStore instance for all enabled feature
 configure.UsePersistence<RavenDB>(_ => _.UseDocumentStoreForTimeouts(...));
 ```
 
-## Wait I thought RavenDB was embedded in NServiceBus?
-
-So a little history
+## History
```
Comment (colin-higgins) [inline README.md]: Do we need this history section?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,32 @@
...
+Add the [NuGet](https://www.nuget.org/packages/NServiceBus.RavenDB/) package to your project.
+Specify `RavenDB` to be used for persistence, and set a RavenDB connection string in your `app.config` as `NServiceBus/Persistence/RavenDB` or pass the connection parameters like this:
 
 ```csharp
-configure.UsePersistence<RavenDB>(_ => _.SetDefaultDocumentStore(new ConnectionParameters
-            {
-                Url = "http://myravendb.mydomain.com/",
-                DatabaseName = "myapp"
-            }));
+configure.UsePersistence<RavenDB>(
```
Comment (colin-higgins) [inline README.md]: This needs updating

File: README.md [hunk 4]
```diff
@@ -32,9 +35,7 @@ Both approaches will use the same DocumentStore instance for all enabled feature
 configure.UsePersistence<RavenDB>(_ => _.UseDocumentStoreForTimeouts(...));
```
Comment (colin-higgins) [inline README.md]: This might need updating

File: README.md (hunk 2 above)
Comment (kbaley) [inline README.md]: Maybe as a separate document. For a readme on how to use the library, it's noise

File: README.md [hunk 5]
```diff
@@ -1,29 +1,32 @@
...
 ======================
 
-Persistence support for NServiceBus RavenDB
+Persistence support for NServiceBus.RavenDB
 
 ## How to use this library
 
-After adding a reference to it from your project, simply specify RavenDB to be used for persistence, and set a RavenDB connection string in your `app.config` as `NServiceBus/Persistence/RavenDB` or pass the conection parameters like this:
+Add the [NuGet](https://www.nuget.org/packages/NServiceBus.RavenDB/) package to your project.
+Specify `RavenDB` to be used for persistence, and set a RavenDB connection string in your `app.config` as `NServiceBus/Persistence/RavenDB` or pass the connection parameters like this:
```
Comment (kbaley) [inline README.md]: Suggestion: Specify `RavenDB` as the persistence mechanism, and set a ...

(In general, I try to go for an active voice in how-to docs.)

File: README.md [hunk 6]
```diff
@@ -46,25 +47,23 @@ This worked but had several negative side effects
  * If consumers of upgraded to newer versions of Raven assemblies, for bug fixes or performance improvements, it was not possible for NServiceBus to leverage these newer assemblies. NServiceBus was hard coded to use the ILMerged versions.
  * Any changes in the compatibility of the Raven Client and Server would require a new version of NServiceBus be release with a new ILMerged version of Raven.
 
-### RavenDB is now resource merged into NServiceBus
+### RavenDB then used to be resource merged into NServiceBus
```
Comment (kbaley) [inline README.md]: Maybe something like:

Iteration 2: Resource merging RavenDB into NServiceBus

**Comment by SimonCropp** [comment]
do we want to go to the effort of maintaining this type of content on every repo or just have a very short blurb and link to http://docs.particular.net/nservicebus/ravendb/

**Comment by colin-higgins** [comment]
@SimonCropp That is fine with me

**Comment by andreasohlund** [comment]
:+1: for the short blurb + link

**Comment by colin-higgins** [comment]
@Particular/tf-async-persistence 
@SimonCropp 
Please review

**Pull request merged**